content/blog/2023/celebrating-our-largest-international-training-event-and-hackathon-to-date.md

```diff
@@ -0,0 +1,102 @@
+
+The hackathon was divided into five focus areas: modules, pipelines, documentation, infrastructure, and subworkflows. We had **411** people register, including **278 in-person attendees** at **16 locations**. This is an increase of **38%** compared to the **289** people that attended our October 2022 event. The hackathon was hosted in multiple countries including Brazil, France, Germany, Italy, Poland, Senegal, Serbia, South Africa, Spain, Sweden, the UK, and the United States.
+
+We would like to thank the many organizations worldwide who provided a venue to host the hackathon and helped make it a resounding success. Besides being an excellent educational event, we resolved many longstanding Nextflow and nf-core issues.
+
+<div style="margin-top: 1rem">
+    <img src="/img/celebrating-our-largest-international-training-event-and-hackathon-to-date-1.jpg" alt="Hackathon photo" />
+</div>
+
+You can access the project reports from each hackathon team over the three-day event compiled in HackMD below:
+
+- [Modules team](https://hackmd.io/A5v4soteQjKywl3UgFa_6g)
+- [Pipelines Team](https://hackmd.io/Bj_MK3ubQWGBD4t0X2KpjA)
+- [Documentation Team](https://hackmd.io/o6AgPTZ7RBGCyZI72O1haA)
+- [Infrastructure Team](https://hackmd.io/uC-mZlEXQy6DaXZdjV6akA)
+- [Subworkflows Team](https://hackmd.io/Udtvj4jASsWLtMgrbTNwBA)
+
+You can also view ten Hackathon videos outlining the event, introducing an overview of the teams, and daily hackathon activities in the [March 2023 nf-core hackathon YouTube playlist](https://www.youtube.com/playlist?list=PL3xpfTVZLcNhfyF_QJIfSslnxRCU817yc). Check out activity in the nf-core hackathon March 2023 Github [issues board](https://github.com/orgs/nf-core/projects/38/views/16?layout=board) for a summary of what each team worked on.
+
+## A diverse and growing community
+
+We were particularly pleased to see the growing diversity of the Nextflow and nf-core community membership, enabled partly by support from the Chan Zuckerberg Initiative Diversity and Inclusion grant and our nf-core mentorship programs. You can learn more about our mentorship efforts and exciting efforts of our global team in Chris Hakkart’s excellent post, [Nextflow and nf-core Mentorship](https://nextflow.io/blog/2023/czi-mentorship-round-2.html) on the Nextflow blog.
+
+The growing diversity of our community was also reflected in the results of our latest Nextflow Community survey, coming soon to the [Seqera Labs blog](https://seqera.io/blog/).
```

Comment:
@olehpratsko if the Seqera blog post goes live first, would be nice to link it directly here..